app/public/fullcalendar-5.5.1/examples/background-events.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
```

Comment:
I noticed that there's this "examples" folder. Is it necessary to keep all of these files?